core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -4020,6 +4031,7 @@ class ReplicaManagerTest {
       doneLatch.countDown()
     } finally {
       replicaManager.shutdown(checkpointHW = false)
+      remoteLogManager.close()
```

Comment:
Can you close/shutdown safely by swallowing any intermediate exceptions occur as mentioned below?
```
      CoreUtils.swallow(replicaManager.shutdown(checkpointHW = false), this)
      CoreUtils.swallow(remoteLogManager.close(), this)
```